docs/ops/detection/ExperimentalDetectronPriorGridGenerator_6.md

```diff
@@ -0,0 +1,128 @@
+## ExperimentalDetectronPriorGridGenerator <a name="ExperimentalDetectronPriorGridGenerator"></a> {#openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6}
+
+**Versioned name**: *ExperimentalDetectronPriorGridGenerator-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronPriorGridGenerator* operation generates prior grids of 
+specified sizes.
+
+**Detailed description**: Operation takes coordinates of centres of boxes and add strides to them to calculate 
+coordinates of prior grids according to next algorithm:
+    
+    for (int ih = 0; ih < layer_height; ++ih)
+        for (int iw = 0; iw < layer_width; ++iw)
+            for (int s = 0; s < number_of_priors; ++s)
+                output_data[0] = priors[4 * s + 0] + step_w * (iw + 0.5)
+                output_data[1] = priors[4 * s + 1] + step_h * (ih + 0.5)
```

Comment:
If we write this C++-code, the we should to use Markdown tags to syntax highlight, that is
```c++
    for (int ih = 0; ih < layer_height; ++ih)
        for (int iw = 0; iw < layer_width; ++iw)
            for (int s = 0; s < number_of_priors; ++s)
                output_data[0] = priors[4 * s + 0] + step_w * (iw + 0.5)
                output_data[1] = priors[4 * s + 1] + step_h * (ih + 0.5)
                output_data[2] = priors[4 * s + 2] + step_w * (iw + 0.5)
                output_data[3] = priors[4 * s + 3] + step_h * (ih + 0.5)
                output_data += 4
```
But, according to @lazarevevgeny, we can write simple description, without C++-code.

Reply:
Let's write simple text description instead of this code.

Reply:
Ok, done.